### PR TITLE
[FIX] mail to partner about creation new client

### PR DESCRIPTION
**Pull request by iledarn**

### CODE REVIEW

File: saas_portal/models/saas_portal.py [hunk 1]
```diff
@@ -551,6 +552,25 @@ def duplicate_database(self, dbname=None, partner_id=None, expiration=None):
                               scope=scope,)[0]
         return url
 
+    @api.multi
+    def send_expiration_info_to_client_db(self):
+        print '\n\n\n'
+        print 'in send_expiration_info_to_client_db'
+        print '\n\n\n'
```
Comment (yelizariev) [inline saas_portal/models/saas_portal.py]: delete prints

File: saas_portal/models/saas_portal.py [hunk 2]
```diff
@@ -439,6 +430,16 @@ class SaasPortalClient(models.Model):
...
+    expiration_datetime_sent = fields.Datetime(help='updates every time send_expiration_info_to_client_db is executed')
+    expiration_changed = fields.Boolean(default=False, help='indicates that expiration_datetime has been changed', compute='_compute_expiration_changed', store=True)
+
+    @api.model
+    @api.depends('expiration_datetime')
+    def _compute_expiration_changed(self):
+        if self.expiration_datetime != self.expiration_datetime_sent:
+            self.expiration_changed = True
+        else:
+            self.expiration_changed = False
```
Comment (yelizariev) [inline saas_portal/models/saas_portal.py]: you can simplify it

File: saas_portal/views/saas_portal.xml [hunk 3]
```diff
@@ -40,6 +40,7 @@
                     <field name="db_storage"/>
                     <field name="trial"/>
                     <field name="expiration_datetime"/>
+                    <field name="expiration_datetime_sent"/>
```
Comment (yelizariev) [inline saas_portal/views/saas_portal.xml]: I think we don't need to show it here

File: saas_portal/views/saas_portal.xml [hunk 4]
```diff
@@ -114,6 +115,7 @@
                 <search string="Search Client">
                     <field name="name" string="Database Name"/>
                     <filter string="In Progress" name="current" domain="[('state', '=','open')]"/>
+                    <filter name="expiration_changed" string="Expiration changed" domain="[('expiration_changed', '=', True)]"/>
```
Comment (yelizariev) [inline saas_portal/views/saas_portal.xml]: delete it. It's only for technical purpose

**Comment by iledarn** [comment]
исправил

**Pull request merged**